account_multicurrency_revaluation/model/res_company.py

```diff
@@ -76,13 +76,13 @@ class ResCompany(models.Model):
     reversable_revaluations = fields.Boolean(
         string="Flag Revaluation Entries",
         help="If marked, the entries created in the process will be flaged "
-        "to be able to filter them out later.",
+        "to be able to filter them out later. ",
```

Comment:
The previous line has space at end already so no need to change this line.
```suggestion
        "to be able to filter them out later.",
```